### DISCUSSION TITLE
Cancelling or dismissing datetime-picker produces error

### DISCUSSION THREAD

**Discussion by beard7** [capawesome-team/capacitor-plugins#224] [Q&A]
How can I tell whether the datetime-picker was cancelled or dismissed?

Thanks

**Comment by robingenz** [comment]
In this case, an error is thrown, see: https://github.com/capawesome-team/capacitor-plugins/blob/7aa2c1681dbf46a4057c9aa056dd095eb80c4e1d/packages/datetime-picker/android/src/main/java/io/capawesome/capacitorjs/plugins/datetimepicker/DatetimePickerPlugin.java#L78
